### PR TITLE
Reduce SSO test logging

### PR DESCRIPTION
**Pull request by c00crane**
Reduce he amount of information that is logged by the SSO OL FATs.
By default now, the logging of TestSettings and Expectations will be OFF.
If we need to debug something, updating the setting of allowPrint in TestSettings will turn logging of both
TestSettings and Expectations on.

### CODE REVIEW

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwtsso_fat,com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.mp.jwt.1.2_fat,com.ibm.ws.security.oidc.client_fat.1,com.ibm.ws.security.oidc.client_fat.2,com.ibm.ws.security.oidc.client_fat.jaxrs,com.ibm.ws.security.oidc.server_fat,com.ibm.ws.security.oidc.server_fat.jaxrs.config,com.ibm.ws.security.oidc.server_fat.oidc,com.ibm.ws.security.social_fat,com.ibm.ws.security.social_fat.LibertyOP,com.ibm.ws.security.social_fat.oidcCertification

File: dev/com.ibm.ws.security.oidc.server_fat/publish/servers/com.ibm.ws.security.openidconnect.server-1.0_fat/configs/server_clientcred_oidc.xml [hunk 1]
```diff
@@ -160,5 +160,12 @@
...
 		actions="connect,resolve" />
-
+	<javaPermission
+		className="java.net.URLPermission"
+		name="https://localhost:8920/oidc/endpoint/OidcConfigSample/token"
+		actions="POST" />
+	<javaPermission
+		className="java.net.URLPermission"
+		name="https://localhost:8920/oauth2tai/snoop"
+		actions="POST" />
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.server_fat/publish/servers/com.ibm.ws.security.openidconnect.server-1.0_fat/configs/server_clientcred_oidc.xml:170]: These changes intended to go in with this PR?

File: dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonMessageTools.java [hunk 2]
```diff
@@ -1,12 +1,12 @@
 /*******************************************************************************
- * Copyright (c) 2020 IBM Corporation and others.
+ * Copyright (c) 2021 IBM Corporation and others.
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonMessageTools.java]: Technically should change to "2020, 2021"

**Review by ayoho — APPROVED** [review]

File: dev/com.ibm.ws.security.oidc.server_fat/publish/servers/com.ibm.ws.security.openidconnect.server-1.0_fat/configs/server_clientcred_oidc.xml (hunk 1 above)
Comment (c00crane) [reply dev/com.ibm.ws.security.oidc.server_fat/publish/servers/com.ibm.ws.security.openidconnect.server-1.0_fat/configs/server_clientcred_oidc.xml:170]: yeah - Kept hitting java2 failures locally - haven't figure out why that sometimes happens, but adding those should not be a problem long term.

File: dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonMessageTools.java (hunk 2 above)
Comment (c00crane) [reply dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonMessageTools.java]: thought I caught all of those - I can update, but, doubt I'll do another full build

**Comment by c00crane** [comment]
#run-libby-bot

**Pull request merged**